src/test/java/shouty/CoordinateTest.java

```diff
@@ -0,0 +1,14 @@
+package shouty;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+
+public class CoordinateTest extends TestCase {
+
+    @Test
+    public void testItCalculatesTheDistanceFromAnotherCoordinateAlongXAxis() {
```

Comment:
Actually his reminds me I struggled to get all the tests to run in the IDE - this was probably a result of me bumbling around with that. I'd welcome someone adding instructions to readme about how to run unit+acceptance tests from an IDE.

Which also reminds me - we need to clean up the readme!